app/src/main/java/org/fossasia/openevent/fragments/LocationsFragment.java

```diff
@@ -61,9 +59,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
             @Override
             public void onRefresh() {
                 DataDownload download = new DataDownload();
-                dbSingleton.clearDatabase(DbContract.Microlocation.TABLE_NAME);
```

Comment:
In api/processor/*, for every success we are clearing the respective tables of database and adding new values.So, for every onRefresh(),its will clear the tables only when we have completed fetching that data else retain it....Other wise it will show blank screen as reported in issue #117